embulk-ruby/lib/embulk/error.rb

```diff
@@ -1,26 +1,24 @@
-
+#
+# As of JRuby 9.3, subclassing a Java classes are full java classes,
+# which introduces some restrictions:
+# See details:
+# https://github.com/jruby/jruby/wiki/CallingJavaFromJRuby#subclassing-a-java-class
+# https://github.com/jruby/jruby/issues/7221
+#
 module Embulk
   # ConfigError is not a ::StandardError but is a java.lang.RuntimeException.
   # "rescue => e" can rescues ConfigError.
   class ConfigError < Java::Config::ConfigException
-    def initialize(message=nil)
-      if message
-        super(message.to_s)
-      else
-        super()
-      end
+    def initialize(*arguments)
```

Comment:
This signature change may mean that `ConfigError` now accepts varieties of arguments, such as: `ConfigError.new(1, 23.4, RuntimeError.new("some error"))`. Is this correct?

Reply:
If then, can you make it rdoc-documented, not for the module `Embulk`, but for these `ClassError` and DataError` classes?